internal/cli/cli.go

```diff
@@ -66,20 +72,24 @@ func Lookup(name string, commands []*Command) (*Command, error) {
 // SetFlags registers a list of functions that configure flags for the command.
 func (c *Command) SetFlags(flagFunctions []func(fs *flag.FlagSet)) {
 	c.flags = flag.NewFlagSet(c.Name, flag.ContinueOnError)
-	c.flags.Usage = constructUsage(c.flags, c.Name)
+	c.flags.Usage = c.usage()
```

Comment:
Just to check my understanding:

- `SetFlags` is called from `init()` in each command's source file
- `SetFlags` calls `usage()` which panics if information is missing
- `init()` will be called automatically (and unconditionally) for all source files within the package
- Therefore we get a panic if any command is missing information, even if that's not the one we're trying to run

I'm pretty confident that's the behavior, and that's what I'd want, but correct me if I'm wrong.